docs/src/pages/guides/localization/Locales.tsx

```diff
@@ -1,7 +1,6 @@
 import * as React from 'react';
+import { useSelector } from 'react-redux';
```

Comment:
It breaks the isolation of the demo, can no longer be opened in codesandbox. IMHO the language of the docs doesn't matter. We translate 5 locales but have a target to support 100 locales with the localization of the library.

Reply:
> can no longer be opened in codesandbox

Why would that be useful? We can disable it.

> We translate 5 locales

But the label is in Chinese, so only supports one of them (and is incorrect 😄 in that one ).

> have a target to support 100 locales with the localization of the library.

These are two different things. This is about localisation (translation) of the docs. However since we don't do it for any other demos, let's revert to English for this one.

Reply:
> Why would that be useful?

To be able to test it in isolation? I had a look at G.A.

- "open in codesandbox" ranks 329 over 517
- "expand" ranks 397 over 533
- "copy source" ranks 392 over 475